scripts/host-check

```diff
@@ -860,7 +911,7 @@ def check_iommu() -> CheckFuncReturn:
             r"pci@([\da-f]{4}:[\da-f]{2}:[\da-f]{2}\.[\da-f])\s+(\S+)", output
         )
         if not matches:
-            return (CheckState.FAILED, "no PCI network devices found")
+            return fail_state, "no PCI network devices found"
```

Comment:
Actually this is a bit out of place... PCI devices are expected to be needed regardless of whether IOMMU is enabled or which driver is used. Maybe just make this an unconditional warning?

Reply:
Should this be in a separate check, rather than part of the IOMMU check? If only igb_uio is enabled, then having a failure/warning about there being no PCI devices associated with the IOMMU check when IOMMU isn't relevant for igb_uio seems a bit weird?

Reply:
Yeah, potentially. Maybe worth discussing offline on how far we want to go with this.

Reply:
Yeah that sounds perfectly reasonable!